File: api-docs/packages/cebcare-api-docs/python/smoke.py
"""Quick client smoke — polite delay on."""

from __future__ import annotations

import json

from cebcare_api_docs import CebcareApiDocsClient


def json_preview(data: object, limit: int = 200) -> str:
    try:
        return json.dumps(data, ensure_ascii=False)[:limit]
    except TypeError:
        return str(data)[:limit]


with CebcareApiDocsClient(default_delay_seconds=1.0) as client:
    print("smoke", client.slug, "->", "demand_mgmt_schedule")
    data = client.demand_mgmt_schedule()
    preview = data[:200] if isinstance(data, str) else json_preview(data)
    print("ok", preview)

File: api-docs/packages/cebcare-api-docs/javascript/examples/smoke.mjs
import { CebcareApiDocsClient } from "../client.mjs";

const client = new CebcareApiDocsClient({ defaultDelayMs: 1000 });
console.log("smoke", CebcareApiDocsClient.slug, "->", "demandMgmtSchedule");
const data = await client.demandMgmtSchedule();
const preview = typeof data === "string" ? data.slice(0, 200) : JSON.stringify(data)?.slice(0, 200);
console.log("ok", preview);
